clients/kafka-common/pom.xml

```diff
@@ -64,7 +64,11 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>io.vertx</groupId>
+      <groupId>org.eclipse.hono</groupId>
+            <artifactId>hono-client</artifactId>
```

Comment:
Exclusions should be applied here like the ones for `hono-core` above (e.g. excluding `vertx-proton`).

Reply:
I have included the exclusions now.